contracts/network/BancorNetwork.sol

```diff
@@ -609,12 +609,9 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
         if (!_networkSettings.isTokenWhitelisted(pool)) {
             revert NotWhitelisted();
         }
+
         // verify pool collection exists and retrieve it
         IPoolCollection poolCollection = _poolCollection(pool);
-        // verify trading is disabled for pool
-        if (poolCollection.tradingEnabled(pool)) {
-            revert TradingEnabled();
```

Comment:
TradingEnabled is unused, can remove the error definition

Reply:
Fixed.